docs/src/logging.md

```diff
@@ -1,43 +1,50 @@
+design facilitates seamless integration with most Kubernetes-compatible log
+management tools, including command line ones like
+[stern](https://github.com/stern/stern).
 
-Each log entry has the following fields:
+!!! Important
+    Long-term storage and management of logs are outside the scope of the
+    operator and should be handled at the Kubernetes infrastructure level.
+    For more information, see the
+    [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/)
+    documentation.
 
-- `level` – Log level (`info`, `notice`, ...).
+Each log entry includes the following fields:
+
+- `level` – The log level (e.g., `info`, `notice`).
 - `ts` – The timestamp (epoch with microseconds).
-- `logger` – The type of the record (for example, `postgres` or `pg_controldata`).
-- `msg` – The actual message or the keyword `record` in case the message is parsed in JSON format.
-- `record` – The actual record with structure that varies depending on the
+- `logger` – The type of log (e.g., `postgres`, `pg_controldata`).
+- `msg` – The log message, or the keyword `record` if the message is in JSON
+  format.
+- `record` – The actual record, with a structure that varies depending on the
   `logger` type.
-- `logging_podName` – The pod where the log was created.
-
-!!! Warning
-    Long-term storage and management of logs is outside the operator's purview,
-    and needs to be provided at the level of the Kubernetes installation.
-    See the
-    [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/)
-    documentation.
+- `logging_pod` – The name of the pod where the log was generated.
 
 !!! Info
-    If your log ingestion system requires it, you can rename the `level` and `ts` field names using the `log-field-level` and
-    `log-field-timestamp` flags of the operator controller. Edit the `Deployment` definition of the
-    `cloudnative-pg` operator.
+    If your log ingestion system requires custom field names, you can rename
+    the `level` and `ts` fields using the `log-field-level` and
+    `log-field-timestamp` flags in the operator controller. This can be configured
+    by editing the `Deployment` definition of the `cloudnative-pg` operator.
 
-## Operator log
+## Operator Logs
 
-You can specify a log level in the cluster spec with the option `logLevel`.
-You can set it to `error`, `warning`, `info`(default), `debug`, or `trace`.
+You can configure the log level for the operator in the cluster specification
+using the `logLevel` option. Available log levels are: `error`, `warning`,
+`info` (default), `debug`, and `trace`.
 
-Currently, you can set the log level only when an instance starts. You can't
-change it at runtime. If you change the value in the cluster spec after the cluster
-was started, it takes effect only in the new pods and not the old ones.
+!!! Important
+   Currently, the log level can only be set at the time the instance starts.
```

Comment:
Looking at the change, and the original is already confusing.
We're talking about clusters and instances in the Operator Logs section.
Going to propose a change here...